app/controllers/admin/api/services_controller.rb

```diff
@@ -46,7 +46,9 @@ def index
   ##~ op.parameters.add :name => " ", :dataType => "custom", :paramType => "query", :allowMultiple => true, :description => "Extra parameters"
   #
   def create
-    service = current_account.create_service(service_params)
+    service = current_account.accessible_services.new
```

Comment:
well using the scope does not change much so I'd say
```suggestion
    service = current_account.build_service
```